package.json

```
@@ -139,6 +141,23 @@
     "test:php": "phpunit",
     "test:php:help": "npm run test:php -- --help"
   },
+  "lint-staged": {
+    "package.json": [
+      "npm run lint:pkg-json"
+    ],
+    "**/*.css": [
+      "npm run lint:css"
+    ],
+    "**/*.js": [
+      "npm run lint:js"
+    ],
+    "**/!(amp).php": [
+      "npm run lint:php"
+    ],
+    "amp.php": [
+      "npm run lint:plugin-bootstrap"
```

This fixes the issue above when linting `amp.php`:

```suggestion
      "vendor/bin/phpcs --runtime-set testVersion 5.2-"
```